OrangeDesignSystemDemo/OrangeDesignSystemDemo/Screens/Modules/Lists/ListModule.swift

```diff
@@ -0,0 +1,170 @@
+//
+// Software Name: Orange Design System (iOS)
+// SPDX-FileCopyrightText: Copyright (c) 2021 - 2023 Orange SA
+// SPDX-License-Identifier: MIT
+//
+// This software is distributed under the MIT license.
+//
+
+import OrangeDesignSystem
+import SwiftUI
+
+/// Show the List style and the Header and Footer of Section. It also
+/// illustrates the editing mode to remove or move items into the list.
+///
+/// 6 types of style are available:
+/// - automatic
+/// - insetGrouped
+/// - grouped
+/// - inset
+/// - plain
+/// - sidebar
+///
+/// As mentioned earlier, SwiftUI will use Inset Grouped style when setting automatic (.automatic) or DefaultListStyle on iOS
+///
+struct ListModule: View {
+
+    // =======================
+    // MARK: Stored Properties
+    // =======================
+
+    private let optionModel: ListModuleOptionsModel
```

Comment:
Use property wrapper __@StateObject__ for both properties instead of just instanciating models in the init.
In this current case objects might be created too many times.
Using **@StateObject** will be better for performances.